client/components/App.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+
+import App from './App';
+import sampleData from '../assets/sampleData';
+
+
+describe('Should mount with data ready to be displayed', () => {
+  const wrapper = shallow(<App sampleData={sampleData} />);
+  it('should have access to an array of data for homes', () => {
+    const span = wrapper.find('span');
+    expect(true).toBe(true);
```

Comment:
Is this line necessary?

Reply:
Yea that was a line left over from the example test. I shall delete it.